graph/oval_graph.py

```diff
@@ -161,75 +161,52 @@ def change_tree_value(self, node_id, value):
     # Methods for interpreting oval tree with SigmaJS
 
     def _get_label(self):
-        return \
-            str(self.node_id).\
-            replace('xccdf_org.ssgproject.content_rule_', '').\
-            replace('oval:ssg-', '').\
-            replace(':def:1', '').\
-            replace(':tst:1', '').\
-            replace('test_', '')
-
-    def _create_node(self, x, y):
-        # print(self.evaluate_tree(),self.value)
+        if self.node_type == 'value':
+            return \
+                str(self.node_id).\
+                replace('xccdf_org.ssgproject.content_rule_', '').\
+                replace('oval:ssg-', '').\
+                replace(':def:1', '').\
+                replace(':tst:1', '').\
+                replace('test_', '')
+        else:
+            if str(self.node_id).startswith(
+                    'xccdf_org.ssgproject.content_rule_'):
+                return \
+                    str(self.node_id).\
+                    replace('xccdf_org.ssgproject.content_rule_', '')
+            return self.value
+
+    def _get_color_node(self):
         if self.value == 'true':
-            return {
-                'id': self.node_id,
-                'label': self._get_label(),
-                'url': 'null',
-                'text': 'null',
-                'title': self.node_id,
-                "x": x,
-                "y": y,
-                "size": 3,
-                "color": '#00ff00'}
+            return '#00ff00'
```

Comment:
It would be better to define a dictionary that maps the value to a color.

For example:
```
VALUE_TO_COLOR = {
   "true" : "#00ff00",
   "false" : "#ff0000",
   "": "#000000"
}
```
And then use the dictionary in a function.


If you use the dictionary approach then all the colors would be defined at a single place, which is always considered better than having them spread across the code.